overview.md

```diff
@@ -0,0 +1,180 @@
+
+## Item Overview
+
+Fundamental to any SpatioTemporal Asset Catalog, an [Item](item-spec/item-spec.md) represents an atomic collection of inseparable 
+data and metadata. A STAC Item is a [GeoJSON](http://geojson.org/) [Feature](https://tools.ietf.org/html/rfc7946#section-3.2)
+and can be easily read by any modern GIS or geospatial library, and it describes a [SpatioTemporal Asset](#what-is-a-spatiotemporal-asset). 
+The STAC [Item JSON specification](item-spec/item-spec.md) includes additional fields for:
+
+* the time the asset represents;
+* a thumbnail for quick browsing;
+* asset links, links to the described data;
+* relationship links, allowing users to traverse other related STAC Items.
+
+A STAC Item can contain additional fields and JSON structures to communicate more information about the
+asset, so it can be easily searched. STAC provides a core set of 
+[Common Metadata](https://github.com/radiantearth/stac-spec/blob/catalog-spec-evolve/item-spec/common-metadata.md)
+and there is a wider community working on a variety of [STAC Extensions](https://github.com/radiantearth/stac-spec/tree/catalog-spec-evolve/extensions) that provide shared metadata for more specific domains. Both aim to describe data with well known, well
+defined terms to enable consistent publishing and better search. For more recommendations on selecting fields
+for an Item see [this section](https://github.com/radiantearth/stac-spec/blob/catalog-spec-evolve/best-practices.md#field-selection-and-metadata-linking) of the best practices document.
+
+### What is a SpatioTemporal Asset
+
+A 'spatiotemporal asset' is any file that represents information about the earth captured in a certain 
+space and time. Examples include Imagery (from satellites, planes and drones), SAR, Point Clouds (from
+LiDAR, Structure from Motion, etc), Data Cubes, Full Motion Video, and data derived from any of those.
+The key is that the GeoJSON is not the actual 'thing', but instead references files and serves as an
+index to the 'assets'. It is [not recommended](best-practices.md#representing-vector-layers-in-stac) 
+to use STAC to refer to traditional vector data layers (shapefile, geopackage) as assets, as they
+don't quite fit conceptually. 
+
+## Catalogs vs Collections
+
+Before we go deep into the Catalogs and Collections, it is worth explaining the relationship 
+between the two and when you might want to use one or the other. 
+
+A Catalog is a very simple construct - it just provides links to Items or to other Catalogs. 
+The closest analog is a folder in a file structure, it is the container for Items, but it can 
+also hold other containers (folders / catalogs). 
+
+The Collection specification extends the Catalog spec, so every Collection can serve as a catalog, 
+requiring the same fields. But it has a number of additional fields: license, extent (spatial and 
+temporal), providers, keywords and summaries. So every Item in the Collection must share those fields, 
```

Comment:
Only the first two fields are requied, the others are optional. "must" implies they are all required.

Reply:
Ok, will tweak. I was just trying to say that if a field is in a collection then it is implicitly true for the item in that collection.